### PR TITLE
New Board Members

### PR DESCRIPTION
**Pull request by ghost**

### CODE REVIEW

**Comment by ozyx** [comment]
@omolazabal Hey Oscar, when you get a chance could you just give a glance to the board members page and double-check that we got everyone? Should be good to merge once that is confirmed.

@PayneGray Nice job on this! The only changes I made were to fill in missing information, and adjust some of the board member images to be square (otherwise the student cards get all stretched out).

File: src/lists/Spring2019List.js [hunk 1]
```diff
@@ -36,13 +36,12 @@ export default [
...
-        image: ChantalleImage,
-        name: "Chantalle Bril",
-        title: "Secretary",
-        description: "Takes meeting minutes and sends out emails. Responsible for all forms of communication.",
-        email: "cbril@csu.fullerton.edu",
-        linkedin: "https://www.linkedin.com/in/chantallebril/",
-        github: "https://github.com/cbril"
+        image: JaneenImage,
+        name: "Janeen Yamak",
+        title: "Comm. Coordinator",
```
Comment (omolazabal) [inline src/lists/Spring2019List.js]: Any reason why Communications was shortened? Everything else is expanded

File: src/lists/Spring2019List.js [hunk 2]
```diff
@@ -133,8 +131,8 @@ export default [
         id: 13,
         image: JesseImage,
         name: "Jesse Mazzella",
-        title: "Web Master",
-        description: "Maintains the official website and manages ACM GitHub repositories. Benevolent web overlord.",
+        title: "Co-WebMaster",
```
Comment (omolazabal) [inline src/lists/Spring2019List.js]: Webmaster*

File: src/lists/Spring2019List.js [hunk 3]
```diff
@@ -143,26 +141,18 @@ export default [
...
-        github: "https://github.com/janeenyamak"
-        
-    },
-    {
+        id: 15,
         image: MadiImage,
         name: "Madison Jordan",
-        description: "",
         title: "Historian",
+        description: "Manages the ACM blog site and writes about recent ACM events.",
```
Comment (omolazabal) [inline src/lists/Spring2019List.js]: "blog" should be changed to "news"

**Comment by omolazabal** [comment]
@ozyx Made some comments. Just minor things. Looks good!

File: src/lists/Spring2019List.js (hunk 1 above)
Comment (ozyx) [reply src/lists/Spring2019List.js]: I was under the assumption that it was so long that it was stretching the student cards out, but looking now that doesn't seem to be the case. Fixed

**Pull request merged**